api/db-cdr/generate-cdr/bq-schemas/ds_linking.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "mode": "REQUIRED",
+    "name": "ID",
+    "type": "INTEGER"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "DENORMALIZED_NAME",
+    "type": "STRING"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "OMOP_SQL",
+    "type": "STRING"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "JOIN_VALUE",
+    "type": "STRING"
+  },
+  {
+    "mode": "REQUIRED",
```

Comment:
Should the database field have nullable false constraint for domain?

Reply:
domain is what the dataset builder uses to lookup the sql chunks. It is invalid to have a null domain.